prep-css-flexbox/align-items.html

```diff
@@ -9,14 +9,14 @@
 
 #card-deck {
 display: flex;
-flex-basis: calc(100% / 3);
+
```

Comment:
Sweet, now we just need `align-items: center` here to vertically center the card deck's child element.

Reply:
![image](https://user-images.githubusercontent.com/31152745/202839709-a4d1d9ec-10d3-4c7f-9ff2-f62c3b630bbf.png)